mavutil.py

```diff
@@ -228,7 +228,7 @@ def post_message(self, msg):
         msg._posted = True
         msg._timestamp = time.time()
         type = msg.get_type()
-        if type != 'HEARTBEAT' or (msg.type != mavlink.MAV_TYPE_GCS and msg.type != mavlink.MAV_TYPE_GIMBAL):
+        if type != 'HEARTBEAT' or (msg.type != mavlink.MAV_TYPE_GCS and msg.type != mavlink.MAV_TYPE_ONBOARD_CONTROLLER and msg.type != mavlink.MAV_TYPE_GIMBAL):
```

Comment:
if type != 'HEARTBEAT' or msg.type not in (mavlink.MAV_TYPE_GCS, mavlink.MAV_TYPE_ONBOARD_CONTROLLER, mavlink.MAV_TYPE_GIMBAL):